index.html

```diff
@@ -0,0 +1,135 @@
+        <li class="item mx-1 "><a href="#" class="nav-link h3">Sponsor</a></li>
+        <li class="item mx-1  "><a href="#" class="nav-link h3">News</a></li>
+
+      </ul> 
+      <div class="hambur my-2">
+        <div class="lin lin1"></div>
+        <div class="lin lin2"></div>
+        <div class="lin lin3"></div>
+      </div> 
+    </div>
+  </section>
+</main>
+<!-- END OF NAV -->
+<!-- Main Section -->
+  <section class="header">
+    <main class="img-cover">
+     <h3 class="h3 text-white-50 container text-center pt-5 pb-1 mt-5">"Connect with like minds"</h3>
+     <div class="headline container py-3 mt-3">
+       <h1 class="h1 headtext">Women In Tech General Summit 2022</h1>
+       <div class="sep mx-3 my-4"></div>
+       <h6 class="venue text-primary mb-3">Dec 12th, 2022, 8pm | Zoom </h6>
+       <a href="#" class="learn py-2 px-2">Learn more</a>
+      </div>
+     <p class="theme h4 pt-5 px-2 text-center">Unite 100 000 Women in Tech to Foster Innovation with Purpose and Impact</p>
+
+    </main>
+  </section>
+<!-- End of Main Section -->
+<!-- Program Schedule -->
+<section class="program container-fluid">
+  <div class="program-box d-flex flex-column justify-content-center align-items-center">
+    <h2 class="h2 text-center pt-5">Summit Program</h2>
+    <div class="pt-1 underprogram"></div>
+  </div>
+  <div class=" d-md-flex flex-row mt-5 container">
+    <div class="program-item py-2 mx-1 mt-3">
+      <div class="program-logo"><i class="fa-sharp fa-solid fa-laptop-file"></i></div>
+      <div class="pro-topic"><h3>Works Exhibition</h3></div>
+      <div class="prosub-topic text-light text-center"><p>A number of Women and groups will have a chance to showcase their products and services in various booths.</p></div>
+    </div>
+    <div class="program-item py-2 mx-1 mt-3">
+      <div class="program-logo"><i class="fa-regular fa-lightbulb"></i></div>
+      <div class="pro-topic"><h3>Innovative session</h3></div>
+      <div class="prosub-topic text-light text-center"><p>The session will be a practical implementation of ideas that result in the introduction of new goods or services or improvement in offering goods or services.</p></div>
+    </div>
+    <div class="program-item py-2 mx-1 mt-3">
+      <div class="program-logo"><i class="fa-regular fa-gift"></i></div>
+      <div class="pro-topic"><h3>Award Presentation</h3></div>
+      <div class="prosub-topic text-light text-center"><p>Participants will get a chance to present their portfolio to sponsors. The winner with the best portfolio will also get $1000 cash price and instant employment</p></div>
+    </div>
+    <div class="program-item py-2 mx-1 mt-3 px-2">
+      <div class="program-logo"><i class="fa-regular fa-people-line"></i></div>
+      <div class="pro-topic"><h3>Interactive Lecture</h3></div>
+      <div class="prosub-topic text-light text-center"><p>Interactive session with leading women in Tech. The Lecture will be conducted in large groups.</p></div>
```

Comment:
- [ ] Comparing your design with the design given, I can see that there is a big difference. Please consider the following comparison 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/198258947-6b750ca2-8781-4607-8491-ccd73d915e7f.png)

**Given design**
![image](https://user-images.githubusercontent.com/90222110/198259074-114894bb-28ed-402f-b7cd-7f43b71a8137.png)

From the design guidelines, you are expected to follow the alignment and layout exactly as it is in the design. Also for the texts, you should use the same color as in the design. **Your cards are aligned in the 2-rows, while on the design given, they are aligned in 1-row. In your design, the cards have big padding inside while in the given design the contents are close together. On the given design, the background is dark while on your design it is white** 
These are a few example differences you can see, therefore I would advise you to improve your styles in order to match the given design very well. this showcase your ability to parse designs 😎